src/events/files.ts

```diff
@@ -13,9 +13,9 @@ export interface FilesEvent {
 export const files = <EventFactory<FilesEvent>> async function (context) {
   const { options } = context.nuxt
 
-  const nuxtIgnore = await pathExists(resolve(options.rootDir, '.nuxtignore'))
```

Comment:
I am not sure if they have difference other than the API style.

According to https://github.com/nodejs/node/issues/39960#issuecomment-913036370, the reason for not having `fs/promises/exists` is because it's not neccessary.